docs/en/Web-Interface.md

```diff
@@ -62,7 +62,7 @@ The configuration page has two sections:
     made to a file not prefixed by a path in the white list. Only whitelisted files will be stored
     in Alluxio.
 
-## Browse File System Page
+## Browse
```

Comment:
you might want to update `Browse File System` to `Browse` on line 67 instead of the header here?